portfolio/src/main/webapp/script.js

```diff
@@ -123,3 +123,13 @@ function showSlides(n) {
   //the dot becomes 'active' (indicator is brighter)
   dots[slideIndex-1].className += " active";
 }
+
+/**
+ * Fetch data from the server.
+ */
+function getServerData() {
+  fetch('/data').then(response => response.text()).then((myJSON) => {
```

Comment:
nit: technically you don't need brackets around this because it's a single parameter but it's good practice to just do it for all arrow functions.

Reply:
I removed curvy brackets and I hope it is okay now